topics/variant-analysis/tutorials/tb-variant-analysis/workflows/tb-variant-analysis.ga

```diff
@@ -1,582 +1,565 @@
 {
     "a_galaxy_workflow": "true",
-    "annotation": "M. tuberculosis Variant Analysis\n",
+    "annotation": "Predict variants and drug resistance from M. tuberculosis sequence samples (Illumina)",
+    "comments": [],
     "creator": [
         {
             "class": "Person",
-            "identifier": "https://orcid.org/0000-0001-6553-5274",
+            "identifier": "0000-0001-6553-5274",
             "name": "Peter van Heusden"
         }
     ],
     "format-version": "0.1",
-    "license": "MIT",
-    "name": "M. tuberculosis Variant Analysis tutorial",
+    "license": "AGPL-3.0-or-later",
```

Comment:
isn't that a strange pick for a workflow license?

Reply:
it's the one I use as well

Reply:
its a web-based resource, thus AGPL instead of GPL?

Reply:
you're welcome to use any you like @pvanheus!

(I imagine to trigger the agpl behaviour it would have to be offered via an API, e.g. planemo running this, or via running in galaxy, but there the workflow is by definition accessible to the user and thus meeting the agpl. other execution systems would need to handle it appropriately as well)

Reply:
I see, fine with me.